arangosh/Shell/V8ClientConnection.cpp

```diff
@@ -66,25 +72,222 @@ V8ClientConnection::V8ClientConnection()
       _vpackOptions(VPackOptions::Defaults) {
   _vpackOptions.buildUnindexedObjects = true;
   _vpackOptions.buildUnindexedArrays = true;
+  /*
   _builder.onFailure([this](fuerte::Error error, std::string const& msg) {
```

Comment:
This pruned the error message set in the application code by an empty message. Imho it doesn't surve any purpose and should kompletely be removed?